gluon/tools.py

```diff
@@ -777,16 +777,16 @@ def encoded_or_raw(text):
                 if attachments:
                     result = mail.send_mail(
                         sender=sender, to=origTo,
-                        subject=unicode(subject), body=unicode(text), html=html,
+                        subject=str(subject), body=str(text), html=html,
```

Comment:
Yeap, I've come to the same solution on my local tree, but totally forgot
to submit a new PR…
I'm sorry for that !

Thanks a lot Raphael for reviving it !

-Mathieu

2016-01-07 17:57 GMT+01:00 Rafael Oliveira Lopes <notifications@github.com>:

> In gluon/tools.py
> <https://github.com/web2py/web2py/pull/1163#discussion_r49096014>:
>
> > @@ -777,16 +777,16 @@ def encoded_or_raw(text):
> >                  if attachments:
> >                      result = mail.send_mail(
> >                          sender=sender, to=origTo,
> > -                        subject=unicode(subject), body=unicode(text), html=html,
> > +                        subject=str(subject), body=str(text), html=html,
>
> Great! I think it's better than matclab's solution.
>
> Will send another commit to the PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/web2py/pull/1163/files#r49096014>.
>